src/v2/Apps/Consign/Routes/SubmissionFlow/ThankYou/ThankYou.tsx

```diff
@@ -2,8 +2,25 @@ import { Button, Flex, Text, Spacer, Box } from "@artsy/palette"
 import { FAQ } from "../../MarketingLanding/Components/FAQ"
 import { SoldRecentlyQueryRenderer } from "../../MarketingLanding/Components/SoldRecently"
 import { RouterLink } from "v2/System/Router/RouterLink"
+import { AnalyticsSchema, useSystemContext, useTracking } from "v2/System"
+import { ContextModule, OwnerType } from "@artsy/cohesion"
+import { useRouter } from "v2/System/Router/useRouter"
 
 export const ThankYou: React.FC = () => {
+  const { user, isLoggedIn } = useSystemContext()
+  const { match } = useRouter()
+  const { trackEvent } = useTracking()
+
+  const trackSubmitAnotherWorkClick = () =>
+    trackEvent({
+      action_type: AnalyticsSchema.ActionType.SubmitAnotherArtwork,
+      context_module: ContextModule.consignSubmissionFlow,
+      context_owner_type: OwnerType.consignmentSubmission,
+      submission_id: match.params.id,
+      user_email: isLoggedIn ? user?.email : "",
```

Comment:
1. **question:** The schema that was requested from Data did not include this field. What will it be used for?
2. **suggestion:** We have access to an anonymous submission's email address at this point. Should we record the email address that was provided on the contact information step for both authenticated and anonymous submissions?

Reply:
1 - The expected schema specified in [this sheet](https://docs.google.com/spreadsheets/d/1QtlyK1XGi1QWSqqqT4mMmZUOYMfPxHlOjupNYZ6LCGo/edit#gid=0) from Data includes the fields `user_email` and `user_id` if user is logged in. I also see it in the body of this event in [Cohesion](https://github.com/artsy/cohesion/blob/main/src/Schema/Events/Consignments.ts#L136). 

2- that's a good suggestion. But the data team might be expecting `user_email` when there is also a `user_id` sent. I'll raise this question in the ticket.

Reply:
We are looking at the same spreadsheet, but different sheets. The sheet titled "Web Submission Confirmation Page" doesn't have the `user_email` field, but the sheet titled "App submission flow" does.

I assume that we'll want it for both, but I'll just confirm in the same ticket.